Yet ANOTHER areas splicing PR

Jolly-66 opened the pull request:
## About The Pull Request
Title.

## Why It's Good For The Game
Hi! Some time last year I was responsible for breaking up the original `space_station_13_areas.dm` ~~and responsible for breaking a few things in the process~~. Anyway- I was looking at one of Ghommies PRs and was snooping around the area files again.

This PR does two things:
1. Nukes `space_station_13_areas.dm` since it was an empty file, just containing a comment that was outdated.
2. Rips apart `areas/station.dm` into `areas/station/*`.

First and foremost, `station.dm` was a whopping `1,771` lines long, and that's just for areas used among our many maps of now and then.

I also deleted `/area/misc/asteroid` because it seemed unused after doing a search.

The old `station.dm` was converted into `areas/station/misc.dm` since not every single area is going to be created with organization in mind.

I also updated comments here and there to be a little more.. obvious? Also and lists I saw were expanded out. They look so much better now!!

## Changelog

N/A this shouldn't really affect anything. Muh organization.

Jolly-66 committed:
wowie, that was a lot of areas

distributivgesetz commented:
my areas are undergoing mitosis again nooo

ZephyrTFA commented on code/game/area/areas/station/cargo.dm:
```diff
@@ -0,0 +1,59 @@
+/*
+* Cargo Areas
+*/
```
get rid of these file headers

timothymtorres commented:
This is great and should help prevent future merge conflicts. +1

Jolly-66 commented:
I'll get back to this in a few days, promise

Jolly-66 committed:
zephyr review + few list touch ups

Jolly-66 committed:
Merge branch 'master' into SplicingUpStationAreas

Jolly-66 commented:
~~press the wrong button I hate Github~~

Addressed Zephyrs review.
However, for Solars and TComms I didn't remove the headers since they're more specific than just the department headers.
Security's header was condensed into a comment (since I never touched bounty code, I left it there).
I looked for a few more lists and expanded them out and added a trailing comma.
